.devcontainer/devcontainer.json

```
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+	"name": "Rust",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/rust:1-1-bookworm",
```

See note below about using a docker file
```suggestion
	"build": {
        // Path is relative to the devcontainer.json file.
        "dockerfile": "Dockerfile"
    },```